### PR TITLE
Introduce sorting by FFT filtered blur detection.

### PR DESCRIPTION
**Pull request by pfakanator**
Blur detection using the mean value of the magnitude spectrum of and inverted FFT.

### CODE REVIEW

File: tools/sort/cli.py [hunk 1]
```diff
@@ -43,14 +43,15 @@ def get_argument_list():
             opts=('-s', '--sort-by'),
             action=Radio,
             type=str,
-            choices=("blur", "face", "face-cnn", "face-cnn-dissim", "face-yaw", "hist",
+            choices=("blur", "fft-blur", "face", "face-cnn", "face-cnn-dissim", "face-yaw", "hist",
```
Comment (torzdf) [inline tools/sort/cli.py:46]: Please change to "blur-fft"

File: tools/sort/sort.py [hunk 2]
```diff
@@ -163,6 +164,18 @@ def sort_blur(self):
...
 
+    def sort_blur_fft(self):
+        """ Sort by fft filtered blur amount with fft"""
+        logger.info("Sorting by estimated fft filtered image blur...")
+
+        fft_blurs = [(filename, self.estimate_blur_fft(image, metadata))
+                     for filename, image, metadata in tqdm(self._loader.load(),
+                                                           desc="Estimating fft filtered blur score",
+                                                           total=self._loader.count,
+                                                           leave=False)]
```
Comment (torzdf) [inline tools/sort/sort.py:175]: To fix the Flake8 line too long nag:
```py
                     for filename, image, metadata in tqdm(
                         self._loader.load(),
                         desc="Estimating fft filtered blur score",
                         total=self._loader.count,
                         leave=False)]
```

**Review by torzdf — CHANGES_REQUESTED** [review]

**Review by pfakanator — COMMENTED** [review]
fixed blur-fft and pep8

**Review by pfakanator — COMMENTED** [review]
PEP8!

**Review by pfakanator — COMMENTED** [review]
fixed flake8 errors

**Review by torzdf — APPROVED** [review]

**Pull request merged**